Project 7 - Bankist/js/script.js

```diff
@@ -0,0 +1,515 @@
+"use strict";
+
+const allUsers = [
```

Comment:
@sknepali Let's create separate file for this and name it `allUsers.js` to organize the code better.

Reply:
Done Sir